### DISCUSSION TITLE
Different Notification Types

### DISCUSSION THREAD

**Discussion by ServerGuyHK** [sct/overseerr#3422] [Q&A]
I have about 15 users on who have access to my Overseer system, most only have three notification settings for Request Approved, Declined and Available. Yet some have a load more and I cannot workout why accounts are different, any ideas ?

Is there a way to set the notification type settings globally so everyone has the same settings ?

**Comment by TheCatLady** [comment]
It's based on user permissions. Users are only able to enable notification types they are eligible to receive based on their permissions. Please read the notification descriptions for details.

**Reply by ServerGuyHK** [reply]
Yes, I fully understand this and I can see it when I edit a user.

The issue here is some users for some reason have more options than others for notifications when you edit a user.

So the first user only has 3 options available
![User 1 - 3 Options](https://user-images.githubusercontent.com/21679728/233754876-cc92f166-d148-4222-888c-627f798093b8.jpg)

The second user has 7 options available
![User 2 - 7 Options](https://user-images.githubusercontent.com/21679728/233754883-7dc114fa-f851-46b0-a76d-cd1d776a6f54.jpg)

I cannot see why there is a difference apart from the users were created at different times.

I only go to notice this when the one with 7 options started to complain that he was getting spammed with email as when his account was imported from Plex, it enabled all of the notification options.
